src/js-form-validator.ts

```diff
@@ -312,8 +320,11 @@ export class FormValidator {
 
       case 'phone':
         if (!allowEmpty && !value.length) return -1;
-        const digits = value.match(/\d+/g);
-        status = digits.length > 0 && /^(?=(?:.{7}|.{10})$)[0-9]*$/.test(digits.join('')) ? 1 : 0;
+        let vals = value.split('x');
+        for (let i = 0, l = vals.length; i < l; i++)
```

Comment:
It seems like this could be done without a for loop. From my quick experiment in the console, it think you could just go `let vals = value.replace(/\D/g, '')`.

Reply:
@samiam236 I checked through this on the changes, and that one slipped by me. that change was not intended to be made.